test/e2e/opm_test.go

```diff
@@ -75,7 +73,7 @@ func buildBundlesWith(containerTool string) error {
 		bundleTag3: bundlePath3,
 	} {
 		if err := inTemporaryBuildContext(func() error {
-			return bundle.BuildFunc(path, "", bundleImage+":"+tag, containerTool, packageName, channels, defaultChannel, false)
+			return bundle.BuildFunc(path, "", bundleImage+":"+tag, containerTool, "", "", "", false)
```

Comment:
Same comment here. Can we have a separate test for this rather than modifying an existing one?